metabase/metabase
# Static pie chart labels

## Changes

Adds data labels to the static pie chart.

## Screenshots

<img width="623" alt="Screen Shot 2022-10-17 at 11 39 51 PM" src="https://user-images.githubusercontent.com/14301985/196267462-4b339a77-9a83-494a-a794-d16b5496d0d1.png">

## How to verify

- Run Maildev/Mailhog and setup email settings on your instance
- Create a pie chart
- Enable "Show data labels" setting
- Add it on a dashboard and send an email subscription
- Ensure static pie chart includes data labels
